enterprise/frontend/src/embedding-sdk/types/auth-config.ts

```diff
@@ -57,3 +57,8 @@ export type MetabaseAuthConfig =
   | MetabaseAuthConfigWithApiKey
   | MetabaseAuthConfigWithJwt
   | MetabaseAuthConfigWithSaml;
+
+export type MetabaseAuthMethod = Exclude<
```

Comment:
Restores the MetabaseAuthMethod type that was deleted by https://github.com/metabase/metabase/pull/58776.